src/sleep/sleep.c

```diff
@@ -262,16 +262,18 @@ static int execute(
                         return log_error_errno(r, "Failed to find location to hibernate to: %m");
                 resume_set = r > 0;
 
+                r = write_efi_hibernate_location(hibernate_location, !resume_set);
                 if (!resume_set) {
+                        if (r == -EOPNOTSUPP)
+                                return log_error_errno(r, "Failed to hibernate as resuming without a valid 'resume=' option won't work");
```

Comment:
Maybe rephrase this as "Refusing hibernation, resuming isn't possible as 'resume=' is not set."?

(Since it's not actually a "failure" for hibernation.)

Reply:
I updated the message with: "No valid 'resume=' option found, refusing to hibernate"